gems/pending/MiqVm/miq_scvmm_vm.rb

```diff
@@ -0,0 +1,85 @@
+require 'MiqVm/MiqVm'
+require 'Scvmm/miq_scvmm_vm_ssa_info'
+
+class MiqScvmmVm < MiqVm
+  def openDisks(disk_files)
```

Comment:
Can't you just use the `openDisks` method in the superclass? Is it only because of the `init_disk_info` usage?
It would be better if the disk, partition, and volume handling remained common.